cics-bundle-maven-site/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Can this file be formatted with 2 spaces instead of tabs like everywhere else? It's tricky to read in a browser as tabs. Happy to review and merge though and you can do that later